pkgs/top-level/beam-packages.nix

```diff
@@ -5,7 +5,7 @@
 , wxGTK32
 , buildPackages
 , stdenv
-, ex_docSupport ? true
+, ex_docSupport ? false
```

Comment:
Supporting `ex_doc` for erlang 27 is more of a pain than it is worth imo.

The most recent error I keep getting is that I need to define `+fnu` or use a utf8 locale.

I would really like to fix this because having the docs locally is nice. However, fixing the build **every** time is frustrating.

Reply:
In theory, we should be able to support this though.